languages/ruby/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,42 @@
+In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
+
+You have four tasks, all related to the time spent cooking the lasasgna.
+
+### 1. Define the expected oven time in minutes
+
+Define the `Lasagna::EXPECTED_MINUTES_IN_OVEN` constant that returns how many minutes the lasagna should be in the oven. According to the cooking book, the expected oven time in minutes is 40:
+
+```ruby
+Lasagna::EXPECTED_MINUTES_IN_OVEN
+# => 40
+```
+
+### 2. Calculate the remaining oven time in minutes
+
+Define the `Lasagna#remaining_minutes_in_oven` method that takes the actual minutes the lasagna has been in the oven as a parameter and returns how many minutes the lasagna still has to remain in the oven, based on the expected oven time in minutes from the previous task.
+
+```ruby
+lasagna = Lasagna.new
+lasagna.remaining_minutes_in_oven(30)
+# => 10
+```
+
+### 3. Calculate the preparation time in minutes
+
+Define the `Lasagna.preperation_time_in_minutes` method that takes the number of layers you added to the lasagna as a parameter and returns how many minutes you spent preparing the lasagna, assuming each layer takes you 2 minutes to prepare.
+
+```ruby
+lasagna = Lasagna.new
+lasagna.preperation_time_in_minutes(2)
+# => 4
+```
+
+### 4. Calculate the total working time in minutes
+
+Define the `Lasagna#total_time_in_minutes` method that takes two named parameters: the first parameter is the number of layers you added to the lasagna, and the second parameter is the number of minutes the lasagna has been in the oven. The function should return how many minutes in total you've worked on cooking the lasagna, which is the sum of the preparation time in minutes, and the time in minutes the lasagna has spent in the oven at the moment.
```

Comment:
Maybe mention the parameter names too?

```suggestion
Define the `Lasagna#total_time_in_minutes` method that takes two named parameters: the first parameter (`number_of_layers`) is the number of layers you added to the lasagna, and the second parameter (`actual_minutes_in_oven`) is the number of minutes the lasagna has been in the oven. The function should return how many minutes in total you've worked on cooking the lasagna, which is the sum of the preparation time in minutes, and the time in minutes the lasagna has spent in the oven at the moment.
```

Reply:
We should probably remove "the first paramater" and just repalce it with the name, as they could actally be defined in either order and it would still work.

Reply:
Right. Updated the suggestion